### ISSUE TITLE
Is it possible to highlight/mark only changed schematic pages and layers?

### ISSUE THREAD

**Issue by frankalicious** [leoheck/kiri#122]
Hello,

I really like this project.  

I have one question / suggestion for improvement.  
Right now I can't see which schematic page or PCB layer has changes between the compared revisions.  
Or did I miss something?  

e.g. The screenshot from the README.  
I have to click through all PCB layers to check which of them has changed.  
![](https://raw.githubusercontent.com/leoheck/kiri/aab3b45ae5a8cc7f5a9df428816ddbdeaf4a366c/misc/kiri_pcb.png)

**Comment by leoheck** [comment]
Hi @frankalicious. 

Kiri has this issue. The way It works it is a bit hard to tell which schematic or layer has changes. It works comparing 2 images, not circuits/layers. Besides that, I was trying to implement something with that regard, but textual changes inside of the schematics/layout don't mean something visible changed. 

However, kiri was made to quick compare pages and layers. This means it was designed to be operated with the keyboard arrows, not with the mouse clicks. Mouse clicks work by convenience. 

If you hit the shortcut `i` you can find all the shortcuts. 

In short, `s` changes the view (schematics/layout). Then you have Arrows `Up/Down` and `Left/Right` to change pages and layers; and to change commits if you need to navigate the commits list.

**Comment by frankalicious** [comment]
> It works by comparing 2 images, not circuits/layers. 

I had a quick look into 2 svgs from the same layer but different revisions.  
You are right. At least the title is different so a simple compare does not work.  

Thanks for the shortcuts.  
Should I close this issue?

**Comment by leoheck** [comment]
I can close It, thanks. If you have something else to say, just reply here or reopen the issue if you can.
